packages/docs/content/guidelines/exceptions.mdx

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
The exemptions pages are now living on Confluence and can be deleted from here! :partying\_face: